docs/extensions/overview.md

```diff
@@ -11,14 +11,13 @@ import MoreHelp from "/src/components/MoreHelp"
 
 Several third-party tools include Semgrep extensions.
 
-
-### Editor
+## Editor
```

Comment:
We should probably split this up somehow, for example:

## Editor

### Official Extensions
- Microsoft Visual Studio Code: [`semgrep-vscode`](https://marketplace.visualstudio.com/items?itemName=semgrep.semgrep)
- Emacs: [`lsp-mode`](https://github.com/emacs-lsp/lsp-mode)

### 3rd party extensions

- IntelliJ IDEA: [`semgrep-idea-plugin`](https://github.com/jtmelton/semgrep-idea-plugin)
- Vim: [`semgrep.vim`](https://github.com/returntocorp/semgrep.vim)

Reply:
For the latter, maybe “Community-contributed extensions?”

Reply:
Let's split them - I agree, to Community-contributed extensions - what do you think @ajbt200128? :)